src/test/kotlin/org/rust/ide/hints/codeVision/RsCodeVisionTestBase.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.hints.codeVision
+
+import com.intellij.codeInsight.codeVision.CodeVisionHost
+import com.intellij.codeInsight.codeVision.CodeVisionInitializer
+import com.intellij.codeInsight.codeVision.settings.CodeVisionSettings
+import com.intellij.codeInsight.codeVision.ui.model.CodeVisionListData
+import com.intellij.codeInsight.codeVision.ui.renderers.CodeVisionRenderer
+import com.intellij.codeInsight.hints.LinearOrderInlayRenderer
+import com.intellij.codeInsight.hints.VcsCodeVisionProvider
+import com.intellij.codeInsight.hints.presentation.PresentationRenderer
+import com.intellij.openapi.editor.Document
+import com.intellij.openapi.editor.EditorCustomElementRenderer
+import com.intellij.openapi.editor.Inlay
+import com.intellij.openapi.util.registry.Registry
+import com.intellij.testFramework.TestModeFlags
+import com.intellij.testFramework.fixtures.CodeInsightTestFixture
+import org.rust.RsTestBase
+import java.util.regex.Pattern
+
+@Suppress("UnstableApiUsage")
+abstract class RsCodeVisionTestBase : RsTestBase() {
```

Comment:
Why can't we just use `CodeVisionTestCase` instead of copying its implementation?